### ISSUE TITLE
[0.3.1] kaffy doesn't compile for elixir < 1.10 due to Kernel.is_struct

### ISSUE THREAD

**Issue by antonioparisi** [aesmail/kaffy#18]
```
...
* Getting kaffy (Hex package)
...
➜  kaffy_test (master) ✗ mix compile
==> kaffy
Compiling 12 files (.ex)

== Compilation error in file lib/kaffy/resource.ex ==
** (CompileError) lib/kaffy/resource.ex:35: undefined function is_struct/1
    (elixir) src/elixir_locals.erl:108: :elixir_locals."-ensure_no_undefined_local/3-lc$^0/1-0-"/2
    (elixir) src/elixir_locals.erl:108: anonymous fn/3 in :elixir_locals.ensure_no_undefined_local/3
    (stdlib) erl_eval.erl:680: :erl_eval.do_apply/6
    (elixir) lib/kernel/parallel_compiler.ex:229: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/7
```

**Comment by aesmail** [comment]
What elixir version are you using?

**Comment by antonioparisi** [comment]
@aesmail 

```
Erlang/OTP 22 [erts-10.4.4] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe]

Elixir 1.9.1 (compiled with Erlang/OTP 20)
```

**Comment by antonioparisi** [comment]
ah @aesmail - I just saw on the elixir 1.10.0 changelog: https://elixir-lang.org/blog/2020/01/27/elixir-v1-10-0-released/

it says:
`
Finally, this release also adds two new guards, is_struct/1 and is_map_key/2, thanks to the strict requirement on Erlang/OTP 21+.
`

I will update my elixir version. ;-)

**Comment by aesmail** [comment]
I'd like to keep this open because I didn't want to only support elixir 1.10+

I know the problem and how to fix it and I might release a patch very soon for this.

Thank you for catching this issue 👍🏼

**Comment by antonioparisi** [comment]
awesome! thanks, will try again later after the new release gets rolled out

**Comment by aesmail** [comment]
This is fixed in the v0.3.2 release.

**Comment by antonioparisi** [comment]
thanks so much @aesmail will try in a bit ;)
